pkg/apply/merge_test.go

```diff
@@ -255,9 +255,12 @@ metadata:
 	err = MergeObjectForUpdate(cur, upd)
 	g.Expect(err).NotTo(HaveOccurred())
 
-	s, ok, err := uns.NestedSlice(upd.Object, "secrets")
+	var s []interface{}
+	var ok bool
+	s, ok, err = uns.NestedSlice(upd.Object, "secrets")
```

Comment:
what does golangci-lint complain about here?

actually, please include the error messages in the commit messages. That makes it possible to confirm that the problem golangci-lint is complaining about is actually a problem, and that the patch actually fixes it rather than just tricking the linter into not noticing the problem.

Reply:
fixed in updated patchset